pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1154,7 +1154,15 @@ self: super: {
 
   xmonad-extras = doJailbreak super.xmonad-extras;
 
-  arbtt = doJailbreak super.arbtt;
+  arbtt = overrideSrc (dontCheck super.arbtt) {
```

Comment:
This is not a good idea. That override will hide all future updates `arbtt` receives. Unless there is some kind of plan how to prevent that from happening, I'd rather not commit a change like that unless there's very good reason to. 

Generally speaking, I'd very much prefer to lobby @nomeata to simply release a new Hackage version.